src/pages/test-report/components/legal-requirements-modal/legal-requirements-modal.ts

```diff
@@ -11,8 +9,7 @@ import { ModalEvent } from '../../test-report.constants';
 })
 export class LegalRequirementsModal {
 
-  // TODO - Change the type to make this generic for other test categories
-  legalRequirements: CatBLegalRequirements;
+  legalRequirements: string[];
```

Comment:
can we use the `legalRequirementsLabels` enum as a type here

Reply:
done